mkdocs.yml

```diff
@@ -168,7 +168,7 @@ nav:
           - 'Firmware update': variants/msi_z690/firmware-update.md
           - 'Recovery': variants/msi_z690/recovery.md
           - 'Hardware Configuration Matrix': variants/msi_z690/hardware-matrix.md
-          - 'Test matrix': variants/msi_z690/test-matrix.md
+          - 'Test matrix': variants/qemu_q35/test-matrix.md
```

Comment:
@Asiderr is it intended?

Reply:
Fixed. dd6b86d09613a308706759b467bc9da4917cc108